File: backend/gym_management_backend/src/main/java/com/gym/entities/Enquiry.java
package com.gym.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "enquiry")
public class Enquiry {
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Id
	@Column(name = "e_id")
	private int eqid;
	@Column(name = "e_fname")
	private String efname;
	@Column(name = "e_lname")
	private String elname;
	@Column(name = "e_email")
	private String eemail;
	@Column(name = "e_phone")
	private String ephone;
	@Column(name = "e_message")
	private String message;
	
	public Enquiry(){
		
	}
	
	public Enquiry(int eqid, String efname, String elname, String eemail, String ephone, String message) {
		super();
		this.eqid = eqid;
		this.efname = efname;
		this.elname = elname;
		this.eemail = eemail;
		this.ephone = ephone;
		this.message = message;
	}

	
	public int getEqid() {
		return eqid;
	}


	public void setEqid(int eqid) {
		this.eqid = eqid;
	}

	public String getEfname() {
		return efname;
	}
	public void setEfname(String efname) {
		this.efname = efname;
	}
	public String getElname() {
		return elname;
	}
	public void setElname(String elname) {
		this.elname = elname;
	}
	public String getEemail() {
		return eemail;
	}
	public void setEemail(String eemail) {
		this.eemail = eemail;
	}
	public String getEphone() {
		return ephone;
	}
	public void setEphone(String ephone) {
		this.ephone = ephone;
	}
	
	public String getMessage() {
		return message;
	}


	public void setMessage(String message) {
		this.message = message;
	}

	@Override
	public String toString() {
		return "Enquiry [eqid=" + eqid + ", efname=" + efname + ", elname=" + elname + ", eemail=" + eemail
				+ ", ephone=" + ephone + ", message=" + message + "]";
	}


	
}

File: frontendNew/adminfrontend/src/Pages/Enquiry.js

import { useState, useEffect } from 'react';
import axios from 'axios';
import { Button } from "react-bootstrap";
import '../css/loginCSS.css';
import '../css/registerform.css';



const Enquiry = () => {

    const[enquiryList,setEnquiryList] = useState([]);
    const [efname, setEfname] = useState("");
    const [elname, setEldesc] = useState("");
    const [eemail, setEemail] = useState(0);
    const [ephone, setEphone] = useState(new Date());
    const[message,setMessage] = useState(0);

    const [lgShow, setLgShow] = useState(false);

    const url = "http://localhost:8080/getallenquiries";
    const getallenquiries = () => {
        axios.get(url).then((response) =>{
            console.log(response.data);
            setEnquiryList(response.data.List);
        })
    }

    const doAlert = (enquiry) => {
        alert("Lol");
        alert(enquiry.eemail);
        window.location.href = "https://mail.google.com/mail/u/1/?ogbl#inbox?compose=new";
      }

    useEffect(()=>{
        getallenquiries();
    },[])

    return (
        <div className="container"> 
        <br></br>       
            <h2>Enquiries</h2>
            {/* <button onClick={getMembers} className="btn btn-primary">Get Members </button> */}
            <table className="table table-striped table-hover">
                <thead Style = "background-color:black ; color : white">
                    <tr>
                        <th>#</th>
                        <th>First Name</th>
                        <th>Last Name</th>
                        <th>Email Id</th>
                        <th>Contact No</th>
                        <th>Message</th>
                        <th>Reply</th>
                    </tr>
                </thead>
                <tbody>
                    {enquiryList.map((enquiry)=>{
                        return (
                        <tr>
                        <td>{enquiry.eqid}</td>
                        <td>{enquiry.efname}</td>
                        <td>{enquiry.elname}</td>
                        <td>{enquiry.eemail}</td>
                        <td>{enquiry.ephone}</td>
                        <td>{enquiry.message}</td>
                        <td> 
                            <Button classname = "shadowify" Style = "background-color : black; color:white">
                            <a href={"mailto:"+enquiry.eemail+"?subject= Reply to your Enquiry about our gym! :- "+enquiry.message}>Reply</a>  
                            </Button> </td>
                        </tr>
                        )
                    })}
                    
                </tbody>
            </table>
            

        </div>
    )
}

export default Enquiry;